docker-compose.yml

```diff
@@ -9,17 +9,7 @@ services:
       context: .
       target: prod
     container_name: branchtaken__hemlock__prod
-    cap_add:
-      - SYS_PTRACE
-    environment:
-      - SSH_AUTH_SOCK
     image: branchtaken/hemlock:prod
-    security_opt:
-      - seccomp:unconfined
-    privileged: true
-    volumes:
-      - .:/home/hemlock/Hemlock
-      - $SSH_AUTH_SOCK:$SSH_AUTH_SOCK
```

Comment:
[Here](https://github.com/cevans87/dotfiles/blob/a1d90017f5be040d5da5a87e1d30e836da908a83/hardlink_home/cevans/docker-compose.yml#L9-L27)'s an example of how I'm replicating this from a docker compose target I defined in my dotfiles.